rpcs3/Emu/system_utils.cpp

```diff
@@ -154,6 +156,95 @@ namespace rpcs3::utils
 		return rap_path;
 	}
 
+	std::string get_c00_unlock_edat_path(const std::string_view& title_id, const std::string_view& content_id)
+	{
+		const std::string home_dir = get_hdd0_dir() + "home";
+
+		std::string edat_path;
+
+		for (auto&& entry : fs::dir(home_dir))
+		{
+			if (entry.is_directory && check_user(entry.name))
+			{
+				edat_path = fmt::format("%s/%s/exdata/%s/%s.edat", home_dir, entry.name, title_id, content_id);
+				if (fs::is_file(edat_path))
+				{
+					return edat_path;
+				}
+			}
+		}
+
+		// Return a sample path tested for logging purposes
+		return edat_path;
+	}
+
+	bool verify_c00_unlock_edat(const std::string_view& title_id, const std::string_view& content_id)
+	{
+		const std::string edat_path = rpcs3::utils::get_c00_unlock_edat_path(title_id, content_id);
+
+		// Check if user has unlock EDAT installed
+		if (!fs::is_file(edat_path))
+		{
+			sys_log.notice(u8"verify_c00_unlock_edat(): “%s” not found", edat_path);
```

Comment:
please use ' everywhere instead of these weird quotations. you can then remove the u8.